### PR TITLE
Add `corrected` argument to std 

### PR DESCRIPTION
**Pull request by pshashk**
Makes `normalise` differentiable

### CODE REVIEW

**Comment by MikeInnes** [comment]
> Makes `normalise` differentiable

Can you add a test for that too? Otherwise looks good.

**Comment by pshashk** [comment]
Maybe we should change `dims` in `normalise` to keyword argument for consistency?

**Comment by MikeInnes** [comment]
I was just thinking the same. It would need a deprecation for the current signature though.

File: src/layers/stateless.jl [hunk 1]
```diff
@@ -40,12 +40,17 @@ but it is more numerically stable.
 logitbinarycrossentropy(logŷ, y) = (1 - y)*logŷ - logσ(logŷ)
 
 """
-    normalise(x::AbstractArray, dims::Int=1)
+    normalise(x::AbstractArray; dims::Int=1)
 
     Normalises x to mean 0 and standard deviation 1, across the dimensions given by dims. Defaults to normalising over columns.
 """
-function normalise(x::AbstractArray, dims::Int=1)
+function normalise(x::AbstractArray; dims::Int=1)
```
Comment (MikeInnes) [inline src/layers/stateless.jl]: I think this type restriction can also be removed; it's fine for us to forward tuples of ints, etc.

File: src/layers/stateless.jl [hunk 2]
```diff
@@ -40,12 +40,17 @@ but it is more numerically stable.
 logitbinarycrossentropy(logŷ, y) = (1 - y)*logŷ - logσ(logŷ)
 
 """
-    normalise(x::AbstractArray, dims::Int=1)
+    normalise(x::AbstractArray; dims::Int=1)
```
Comment (MikeInnes) [inline src/layers/stateless.jl]: Watch out for the docstring also

**Comment by MikeInnes** [comment]
Thanks a lot!

**Pull request merged**